test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -1227,11 +1226,13 @@ public void TensorFlowSentimentClassificationTest()
             // For explanation on how was the `sentiment_model` created 
             // c.f. https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model/README.md
             string modelLocation = @"sentiment_model";
-            using var pipelineModel = _mlContext.Model.LoadTensorFlowModel(modelLocation).ScoreTensorFlowModel(new[] { "Prediction/Softmax" }, new[] { "Features" })
+            using var pipelineModel = _mlContext.Model.LoadTensorFlowModel(modelLocation, false).ScoreTensorFlowModel(new[] { "Prediction/Softmax" }, new[] { "Features" })
                 .Append(_mlContext.Transforms.CopyColumns("Prediction", "Prediction/Softmax"))
                 .Fit(dataView);
             using var tfEnginePipe = _mlContext.Model.CreatePredictionEngine<TensorFlowSentiment, TensorFlowSentiment>(pipelineModel);
 
+            var schema = pipelineModel.GetOutputSchema(dataView.Schema);
```

Comment:
Should we do something with this `schema`? Can we assert something about it?

Reply:
Actually, editing that test was an error on my part. Let me revert the changes I made to that test method. The only test that should be changed is `TreatOutputAsBatched`, and that one does have asserts on the schema checking the changes I made.

Reply:
Changes have been reverted, its now back to how it originally was.